andela/ah-orcas-frontend
# #161254622 Rate articles on Authors Heaven

#### What does this PR do?
```This PR merges the rate article branch with develop ```
#### Description of Task to be completed?
```allow logged in users to rate articles on Authors Haven```
```allow users to view average rating of an article ```
```Redesign the view article UI```
#### How should this be manually tested?
```Clone this repository```
- git clone https://github.com/andela/ah-orcas-frontend.git
```Change the working directory to the project folder```
- cd ah-orcas-frontend
```Checkout to the appropriate branch```
- git checkout ft-rate-an-article-161254622
```Install the required dependencies and start the server```
- npm install
- npm start
```You should be able to see the star ratings when you view a single article, i.e. average rating. Also, you will be able to rate a given article provided you are logged in```
#### Any background context you want to provide?
```Having ratings on a given article helps in getting the statistics that can be useful when collecting data on which topic is loved by the users ```
#### What are the relevant pivotal tracker stories?
[#161254622](https://www.pivotaltracker.com/story/show/161254622)
#### Screenshots (if appropriate)
<img width="1358" alt="screenshot 2018-12-20 at 13 31 27" src="https://user-images.githubusercontent.com/11458891/50283217-3ee6d000-0466-11e9-8809-e5616c3f7e77.png">

<img width="1343" alt="screenshot 2018-12-20 at 14 49 40" src="https://user-images.githubusercontent.com/11458891/50283293-82413e80-0466-11e9-922d-c85f62c0cd26.png">


#### Checklist:
